QuizApp/QuizApp/UI/Screens/Login/LoginView.swift

```diff
@@ -7,7 +7,7 @@ struct LoginView: View {
     var body: some View {
         VStack {
             Text("PopQuiz")
-                .font(.title)
+                .font(.heading1)
                 .fontWeight(.bold)
```

Comment:
Please check if you need this because heading1 is defined as `Font.custom("SourceSansPro-Bold", size: 32)`